ShinyExpenseTracker/src/ca/ualberta/cs/shinyexpensetracker/models/DestinationList.java

```diff
@@ -0,0 +1,67 @@
+package ca.ualberta.cs.shinyexpensetracker.models;
+
+import java.util.ArrayList;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class DestinationList extends Model<DestinationList> {
```

Comment:
If `ExpenseClaim` just has an `ArrayList<Destination>` to hold its child destinations, what is this class for?